jme3-core/src/main/java/com/jme3/asset/ImplHandler.java

```diff
@@ -124,10 +124,7 @@ protected T initialValue() {
             } catch (InstantiationException | IllegalAccessException
                     | IllegalArgumentException | InvocationTargetException
                     | NoSuchMethodException | SecurityException ex) {
-                logger.log(Level.SEVERE, "Cannot create locator of type {0}, does"
-                        + " the class have an empty and publicly accessible"
-                        + " constructor?", type.getName());
-                logger.throwing(type.getName(), "<init>", ex);
```

Comment:
Hi @codex128 ,
Why did u remove the logger.throwing? is it OK that no exception will be thrown at that point and a null will be returned instead?

Reply:
@codex128 ,
Maybe I'm missing something but I see that the code you deleted was also logging a severe exception so what is the significant change you made here?
Also, you deleted the logger.throwing line so I guess you find out that logging the exception object is not clear enough so you removed it right?
Last question - why not using logger.error instead of logger.info? 

10X